test/torchaudio_unittest/datasets/fluentcommands_test.py

```diff
@@ -0,0 +1,145 @@
+import csv
+import os
+import random
+import string
+from pathlib import Path
+
+from torchaudio.datasets import fluentcommands
+from torchaudio_unittest.common_utils import (
+    get_whitenoise,
+    save_wav,
+    TempDirMixin,
+    TorchaudioTestCase,
+)
+
+HEADER = ["", "path", "speakerId", "transcription", "action", "object", "location"]
```

Comment:
Is the `speakerId` from original csv? Or can we change it to
```suggestion
HEADER = ["", "path", "speaker_id", "transcription", "action", "object", "location"]
```

Reply:
yea it is from the original csv